frontend/src/i18n/translations/en.json

```diff
@@ -40,5 +40,9 @@
   "profile.company-info": "Company Information",
   "profile.my-info": "My Information",
   "profile.user-management": "User Management",
-  "profile.payment-info": "Payment Information"
+  "profile.payment-info": "Payment Information",
```

Comment:
We're not really using i18n and localization in onroutebc at the moment, but I guess you could leave this in. It's just that we're not gonna use it.

Reply:
gotcha - thanks. Removed these references.